packages/backend-for-frontend/src/services/catalogService.ts

```diff
@@ -578,6 +578,22 @@ export function catalogServiceBuilder(
         },
       });
     },
+
+    updateSignalHubFlag: async (
```

Comment:
Align with operationId in spec and with other methods that `updateEServiceSomething`

```suggestion
    updateEServiceSignalHubFlag: async (
```